### PR TITLE
Change list to only show upcoming events

### PR DESCRIPTION
**Pull request by kwunkiuma**
Listing all events will only show events that are occurring in the future.

Events are also ordered chronologically

### CODE REVIEW

File: DiscordBot/Commands/EventCommands.cs [hunk 1]
```diff
@@ -279,20 +279,24 @@ public async Task ListEvents(CommandContext context)
         {
             var eventsList = await eventsSheetsService.ListEventsAsync();
 
+            var sortedEventsList = eventsList
+                .Where(x => x.Time > DateTime.Now)
```
Comment (kwunkiuma) [inline DiscordBot/Commands/EventCommands.cs:283]: Might it be better to show events that happened earlier on the same day?

**Review by kwunkiuma — COMMENTED** [review]
Also have one question.

File: DiscordBot/Commands/EventCommands.cs (hunk 1 above)
Comment (FridaTveit) [reply DiscordBot/Commands/EventCommands.cs:283]: No, I think all upcoming events is the most useful. It's always possible to add a separate command for listing all events at some point if necessary 🙂

**Review by FridaTveit — COMMENTED** [review]
Looks good 🙂

**Pull request merged**